### PR TITLE
Product list

### PR DESCRIPTION
**Pull request by riyas2516**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -11702,4 +11702,300 @@
...
+}
+/* .enquiry-box-container{
+	margin: auto;
+} */
+.enquiry-section .enquiry-box-container .enquiry-content h2{
+    margin: 24px 0;
+	color: #ff0000;
+	text-align: center;
+}
+/* .enquiry-content form{
```
Comment (projectsznz) [inline css/style.css]: @riyas2516 remove commented lines, if they are not required

File: css/style.css [hunk 2]
```diff
@@ -11702,4 +11702,300 @@
...
+}
+.enquiry-section{
+	width: 100%;
+	height: 100vh;
+	background-color: #02022b;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+}
+/* .enquiry-box-container{
```
Comment (projectsznz) [inline css/style.css]: @riyas2516 remove commented lines, if they are not required

File: css/style.css [hunk 3]
```diff
@@ -11702,4 +11702,300 @@
...
+	left: 2px;
+}
+.FM_m20:hover { transform: rotate(-15deg) scale(1.3); }
+.mb-2{
+    margin-top: 30px;
+    font-size: 16px;
+    font-family: Arial, Helvetica, sans-serif;
+    line-height: 2px;
+	color: white;
+	/* margin-left: 10px; */
```
Comment (projectsznz) [inline css/style.css]: @riyas2516 same here, remove commented lines

File: css/style.css [hunk 4]
```diff
@@ -11702,4 +11702,300 @@
...
+}
+ .ms-3{
+    color: white;
+	margin-left: 10px;
+}
+.review-date{
+	float: left;
+	color: white;
+	font-size: 12px;
+    /* margin-left: 210px; */
```
Comment (projectsznz) [inline css/style.css]: same here, remove commented lines

File: css/style.css [hunk 5]
```diff
@@ -11702,4 +11702,300 @@
...
+	border-radius: 5px;
+	color: white;
+	background-color: #ff0000;
+	cursor: pointer;
+	font-weight:600;
+	letter-spacing: 1px;
+	
+}
+.user-review-more-btn button:hover{
+	/* color: #ff0000; */
```
Comment (projectsznz) [inline css/style.css]: remove

**Review by projectsznz — CHANGES_REQUESTED** [review]

**Comment by projectsznz** [comment]
@riyas2516 change the name of file "copy of DSC01266.jpg" to just DSC01266.jpg

**Comment by projectsznz** [comment]
@riyas2516 remove this duplicate image [images/DSC01163 (1).jpg](https://github.com/projectsznz/gj-brushes/pull/13/files#diff-2bf7d21739ee0f46950228e8149d94b17b77796d366b3ea38712a8869d808758)

**Comment by riyas2516** [comment]
> @riyas2516 remove this duplicate image [images/DSC01163 (1).jpg](https://github.com/projectsznz/gj-brushes/pull/13/files#diff-2bf7d21739ee0f46950228e8149d94b17b77796d366b3ea38712a8869d808758)

@SaravanaZen finished all corrections you mentioned

**Pull request merged**